webpack.config.js

```diff
@@ -66,7 +67,23 @@ module.exports = (env, argv) => {
                         {
                             loader: 'style-loader',
                             options: {
-                                insertAt: 'top'
+                                insert: function insertAtTop(element) {
```

Comment:
What is this change about? Do we even use style-loader / css-loader in this repo?

Reply:
Hmm.. good question, this may have been added as a copy-paste from the DCC configuration a long time ago. In DCC this is used to guarantee styles from one of the third-party components (don't remember which) are applied correctly.

Reply:
Comes from https://github.com/plotly/dash-html-components/commit/909ee5894ccfea98c032bcc048b8e09b3797bd01#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fR70 -- I see no reason to keep this, will 🔪